index.html

```diff
@@ -7,8 +7,15 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
 
-    <!-- Main CSS style -->
+    <!-- CSS styles -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
+      crossorigin="anonymous"
+    />
```

Comment:
Considering that Bootstrap 5 is in beta, shouldn't we choose Boostrap 4 for now?

Reply:
I deleted the other comment by mistake, but maybe is best to stick with 4 like you suggest